tests/lib/snaps/gnome-software/bin/gnome-software

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+echo "Open URI: $1"
```

Comment:
Heh, nice idea :)

Reply:
Something like:
```
Fake snap got %s
```
might help make the test more clear. As it is the test doesn't make clear that Open URI comes from this place. In other words,  the diff is clear as we have everything here, but someone reading the test in a short while won't realize what's actually printing "Open URI".